package-lock.json

```diff
@@ -0,0 +1,17944 @@
+{
```

Comment:
We are using yarn on this project. Would it be possible to delete this file and update the yarn lock file instead?